src/modules/Todo.js

```diff
@@ -0,0 +1,6 @@
+export default class Todo {
+  constructor(disc, completed) {
```

Comment:
- [ ] [REQUIRED] Kindly add the property `index` and ensure the property index is set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal 5) as stated in the project requirement below:
<img width="910" alt="Screenshot 2022-12-16 at 19 36 56" src="https://user-images.githubusercontent.com/58692590/208166326-c2836929-db99-4632-bd9b-7876839b6331.png">